requirements.yml

```
@@ -4,22 +4,20 @@
   version: 1.1.0
 
 - src: ome.ice
-  version: 3.0.0
+  version: 3.1.0
 
 - src: ome.java
   version: 2.0.3
 
-- src: ome.omego
-  version: 0.1.3
-
 - src: ome.omero_common
   version: 0.3.2
 
 - src: ome.omero_python_deps
   version: 2.0.1
 
-- src: ome.omero_server
-  version: 2.1.0
+- name: ome.omero_server
+  src: https://github.com/ome/ansible-role-omero-server/archive/06bd02493b0afd5ee2c74f773fbb08b550ed82a2.tar.gz
```

```suggestion
  src: https://github.com/ome/ansible-role-omero-server/archive/85014735233254a1bbaa4c455a84cfa5391204a4.tar.gz
```
If travis passes https://travis-ci.org/ome/ansible-role-omero-server/builds/611504262